README.md

```diff
@@ -1 +1,195 @@
+
+- Clone the repository to your local machine using the git clone command as described in the previous answer.
+Navigate to the project folder using a command-line interface such as Terminal or Command Prompt.
+Install any necessary dependencies using a package manager such as NPM or Yarn. You can do this by running the npm install or yarn install command in the project folder.
+Start the local server using a command such as npm start or yarn start, depending on the project's setup.
+Open a web browser and navigate to the local server's URL to view and test the project.
+-
+
+### Usage
+
+- Assuming that you have installed the project's dependencies and started the local server, you can run the project by following these steps:
+
+Open a web browser such as Google Chrome, Mozilla Firefox, or Safari.
+In the address bar, type the URL of the local server that you started in the previous step. This will typically be something like http://localhost:3000.
+Press enter to navigate to the local server's URL.
+The project should now be running in your web browser. Use the website's interface to interact with the project and test its functionality.
+
+### Run tests
+
+To run tests, run the following command:
+
+- To run tests run :
+- npx hint .
+- npx stylelint "\*_/_.{css,scss}"
+- modules
+
+### Deployment
+
+- Project is deployed using [GitHub Pages](https://github.com/techmoves/Awesome-books-es6.git)
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Phelelani Ndlovu**
+
+- GitHub: [@techmoves](https://github.com/techmoves)
+- Twitter: @Dopezonke_
+- LinkedIn: https://www.linkedin.com/in/phelelani-ndlovu-883916160/
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features to be added:
+
+- [X] **[booklist in collection]**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Understand the project's goals: Before I start contributing to a project, it's important to understand its goals, scope, and target audience. Read the project's documentation and review its codebase to get a sense of what it does and how it works.
+
+Create a new issue: If you I find an existing issue that matches your contribution, consider creating a new one. sure about the problem or feature request clearly and provide any relevant details or context.
+
+
+
+Contributions, issues, and feature requests are welcome!
+
+
+
+Feel free to check the [issues page](https://github.com/techmoves/Awesome-books-es6.git)
+    
+<p align="right">(<a href="">back to top</a>)</p>
+
+<!-- SUPPORT --> 
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+(back to top)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
```

Comment:
Kindly add a message acknowledging whoever inspired this project🙏

Reply:
resolved